config/app.default.php

```diff
@@ -236,7 +236,7 @@
             'timezone' => 'UTC',
             'flags' => [],
             'cacheMetadata' => true,
-            'log' => false,
+            'log' => true,
```

Comment:
While I like and debug kit for many things, I prefer to see the sql logging response right away on a secondary screen when working with the query builder; also this is useful when working with Postman.app where we don't have DebugKit - in the very long run (4.0, 5.0) it would be nice if DebugKit had a remote connection option.

Anyway I will remove it from this PR.